api/src/main/java/org/pmiops/workbench/firecloud/FirecloudTransforms.java

```diff
@@ -25,8 +25,8 @@ public static Map<String, RawlsWorkspaceAccessEntry> extractAclResponse(
   }
 
   public static RawlsWorkspaceACLUpdate buildAclUpdate(
-      String email, WorkspaceAccessLevel updatedAccess) {
-    RawlsWorkspaceACLUpdate update = new RawlsWorkspaceACLUpdate().email(email);
+      String username, WorkspaceAccessLevel updatedAccess) {
+    RawlsWorkspaceACLUpdate update = new RawlsWorkspaceACLUpdate().email(username);
```

Comment:
can't update the name of `RawlsWorkspaceACLUpdate.email` because that's defined by the Rawls API